src/Symfony/Component/ErrorHandler/Command/ErrorDumpCommand.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ErrorHandler\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\ErrorHandler\ErrorRenderer\ErrorRendererInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+use Symfony\WebpackEncoreBundle\Asset\EntrypointLookupInterface;
+
+/**
+ * Dump error pages to plain HTML files that can be directly served by a web server.
+ *
+ * @author Loïck Piera <pyrech@gmail.com>
+ */
+#[AsCommand(
+    name: 'error:dump',
+    description: 'Dump error pages to plain HTML files that can be directly served by a web server',
+)]
+final class ErrorDumpCommand extends Command
+{
+    public function __construct(
+        private readonly string $defaultPath,
+        private readonly Filesystem $filesystem,
+        private readonly ErrorRendererInterface $errorRenderer,
+        private readonly ?EntrypointLookupInterface $entrypointLookup = null,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument('status-codes', InputArgument::IS_ARRAY, 'Status codes to dump error pages for')
+            ->addOption('path', null, InputOption::VALUE_REQUIRED, 'Path where to dump the error pages in', $this->defaultPath)
+            ->addOption('clear', null, InputOption::VALUE_NONE, 'Remove the directory before dumping new error pages')
```

Comment:
Instead, I would ask confirmation when overriding a directory with existing files, or use the classic `--force/-f` to force the removal of an existing directory.